docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -92,63 +95,168 @@ You can also build the collector-contrib repository by cloning it and running:
+<2> At a minimum, you must define the URL of the APM Server instance you are sending data to. See the <<open-telemetry-elastic-config-ref,configuration reference>>
+for additional configuration options, like specifying an API key, secret token, or TLS settings.
+<3> To translate metrics, the Elastic exporter must be defined in `service.pipelines.metrics.exporters`. 
+<4> To translate trace data, the Elastic exporter must be defined in `service.pipelines.traces.exporters`.
 
 NOTE: For more information about getting started with an OpenTelemetry Collector,
 see the {ot-collector}[OpenTelemetry collector] docs.
 
+[float]
+[[open-telemetry-elastic-caveats]]
+==== Caveats
+
+If collecting metrics, please note that the https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-api/latest/io/opentelemetry/api/metrics/DoubleValueRecorder.html[`DoubleValueRecorder`]
+and https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-api/latest/io/opentelemetry/api/metrics/LongValueObserver.html[`LongValueRecorder`] metrics are not yet supported.
+
+[[open-telemetry-elastic-verify]]
+==== Verify OpenTelemetry metrics data
+
+Use *Discover* to validate that metrics are successfully being reported to {kib}.
+
+. Launch {kib}:
++
+--
+include::../tab-widgets/open-kibana-widget.asciidoc[]
+--
+
+. In the side navigation, under *{kib}*, select *Discover*.
+. Select `apm-*` as your index pattern.
+. Filter the data to only show documents with metrics: `processor.name :"metric"`
+. Narrow your search with a known OpenTelemetry field. For example, if you have an `order_value` field, add `order_value: *` to your search to return
+only OpenTelemetry metrics documents.
+
+[[open-telemetry-elastic-kibana]]
+==== Visualize in {kib}
+
+TSVB within {kib} is the recommended visualization for OpenTelemetry metrics. TSVB is a time series data visualizer that allows you to use the full power of the
+{es} aggregation framework. With TSVB, you can combine an infinite number of aggregations to display complex data.
+
+In this example eCommerce OpenTelemetry dashboard, there are four visualizations; sales, order count, product cache, and system load. The dashboard provides us with business
+KPI metrics, along with performance-related metrics.
+
+[role="screenshot"]
+image::images/ecommerce-dashboard.png[OpenTelemetry visualizations]
+
+Let's have a look at how this dashboard was created, specifically the Sales USD and System load visualizations.
+
+. In the side navigation, under *{kib}*, select *Dashboard*.
+. Click *Dashboard* and select *Create dashboard*.
+. Click *Save*, enter the name of your dashboard, and then click *Save* again.
+. Let’s add a Sales USD visualization. Click *Edit*.
+. Click *Create new* and then select *TSVB*.
+. For the label name, enter Sales USD, and then select the following:
++
+* Aggregation: `Positive Rate`.
+* Field: `order_sum`.
+* Scale: `auto`.
+* Group by: `Everything`
+. Click *Save and return*.
```

Comment:
My button was only titled "Save". I also had to title the viz at this point. Only after that could I click Save and return